apps/browser/src/pages/Dojo.tsx

```diff
@@ -201,7 +201,7 @@ function Dojo() {
         userWorkspace,
         scripts,
         env,
-        [DEV_AGENT_CONFIG, RESEARCH_AGENT_CONFIG]
+        [DEVELOPER_AGENT_CONFIG, RESEARCHER_AGENT_CONFIG]
```

Comment:
I wish this was a default in evo, at least for the hackathon
I find it a bit tedious to keep changing Dojo and CLI each time we change something in Evo

Reply:
This has been addressed.